src/frontend/screens/Settings/ProjectConfig/ManagePeople.tsx

```diff
@@ -0,0 +1,157 @@
+import * as React from "react";
+import { StyleSheet, View } from "react-native";
+import { useNavigation } from "react-navigation-hooks";
+import { getUniqueId } from "react-native-device-info";
+import MaterialIcon from "react-native-vector-icons/MaterialIcons";
+import { FormattedMessage, defineMessages } from "react-intl";
+
+import { MAPEO_BLUE } from "../../../lib/styles";
+import { showWipAlert } from "../../../lib/utils";
+import Button from "../../../sharedComponents/Button";
+import Text from "../../../sharedComponents/Text";
+
+import { MemberRow } from "./MemberRow";
+
+const m = defineMessages({
+  managePeople: {
+    id: "screens.Settings.ManagePeople.managePeople",
+    defaultMessage: "Manage People",
+  },
+  add: {
+    id: "screens.Settings.ManagePeople.add",
+    defaultMessage: "Add",
+  },
+  leaveProject: {
+    id: "screens.Settings.ManagePeople.leaveProject",
+    defaultMessage: "Leave Project",
+  },
+  remove: {
+    id: "screens.Settings.ManagePeople.remove",
+    defaultMessage: "Remove",
+  },
+  yourDevice: {
+    id: "screens.Settings.ManagePeople.yourDevice",
+    defaultMessage: "(You) {deviceName}",
+  },
+});
+
+interface MemberDevice {
+  id: string;
+  name: string;
+}
+
+interface Props {
+  loading: boolean;
+}
+
+export const ManagePeople = ({ loading }: Props) => {
+  const navigation = useNavigation();
+
+  // TODO: dummy state
+  const [members] = React.useState<MemberDevice[]>([
+    { id: "1", name: "Android Device Example A" },
+    { id: "2", name: "Android Device Example B" },
+    { id: "3", name: "Android Device Example C" },
+  ]);
```

Comment:
open to removing this before merging since it's mostly just for demonstrative purposes

Reply:
Lets keep it in here for now, until we actually do add backend functionality. With the WIP and "Device Example" It would be hard to confuse this as real data